StardewArchipelago/Locations/CodeInjections/Vanilla/BackpackInjections.cs

```diff
@@ -112,6 +123,13 @@ private static void BuyBackPackArchipelago(GameLocation __instance, out bool __r
                         responseDontPurchase
                     }, "Backpack");
             }
+            else if (_archipelago.SlotData.Mods.HasMod(ModNames.BIGGER_BACKPACK) & _locationChecker.IsLocationNotChecked(PREMIUM_PACK)
+            && _archipelago.GetReceivedItemCount("Progressive Backpack") >= 2)
+            {
+                var backpackType = AccessTools.TypeByName("BiggerBackpack.Mod");
+                var clickBackpackMethod = ModEntry.Instance.Helper.Reflection.GetMethod(backpackType, "clickBackpack");

```

Comment:
The original code from that is literally just hard coding an english string and creating a dialogue, we could just do the same as above and avoid depending on their code for that